ale_linters/markdown/remark_lint.vim

```diff
@@ -0,0 +1,30 @@
+" Author rhysd https://rhysd.github.io/
+" Description: remark-lint for Markdown files
+
+function! ale_linters#markdown#remark_lint#Handle(buffer, lines) abort
+    " matches: '  1:4  warning  Incorrect list-item indent: add 1 space  list-item-indent  remark-lint'
+    let l:pattern = '^ \+\(\d\+\):\(\d\+\)  \(warning\|error\)  \(.\+\)$'
+    let l:output = []
+
+    call writefile(a:lines, $HOME . '/this_is_test.txt')
```

Comment:
You left a test line here.

My recommended way to debug jobs and channels is to use `ch_logfile` in Vim 8.

Reply:
ah, sorry...